.github/workflows/trivy.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow performs trivy check of docker config files.
+
+name: Trivy
+
+on:
+  pull_request:
+    paths:
+      - 'devops/containers/**'
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Trivy
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          sparse-checkout: devops
+
+      # There is a github action, but for some reason it ignores ignore-file.
+      - name: Install Trivy
+        run: |
+          curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh
+          ./bin/trivy --version
+
+      - name: Run Trivy vulnerability scanner
+        run: ./bin/trivy config devops/containers --ignorefile=devops/.trivyignore.yaml
```

Comment:
[Must have] We will need a report file produced by the tool to be able to use it as an evidence of a completed scan. Therefore, this job should probably save an artifact. I do not expect much changes in it, so keeping it for a 1-3 days should be more than enough for us to run the workflow on a target branch and then grab that report.

[Nice to have] Would be cool to post results of the scan under `Security` tab like we have for Bandit scan

Reply:
1. [d6e46dc](https://github.com/intel/llvm/pull/16840/commits/d6e46dcfe5e82fc9c9f4fdf6d7c2a6c4e375a664)
2. That's what I made initially, but then we [switched](https://github.com/intel/llvm/pull/16840/commits/55d424cd48ad3a940b77ad653d94175d96c3f1c0) the workflow to trigger on pull request. We may also add a nightly run to post the result to security, but I guess it would be excessive.